tuned/utils/config_parser.py

```diff
@@ -0,0 +1,43 @@
+# ConfigParser wrapper providing compatibility layer for python 2.7/3
+
+try:
+	python3 = True
+	import configparser as cp
+except ImportError:
+	python3 = False
+	import ConfigParser as cp
+	from StringIO import StringIO
+	import re
+
+class Error(cp.Error):
+	pass
+
+if python3:
+
+	class ConfigParser(cp.ConfigParser):
+		pass
+
+else:
+
+	class ConfigParser(cp.ConfigParser):
+
+		def __init__(self, delimiters=None, inline_comment_prefixes=None, strict=True, *args, **kwargs):
```

Comment:
Am I blind or you don't use `delimiters` anywhere? This way this does not resolve #378

Reply:
@yarda You don't use `delimiters` and `strict` anywhere so it even get around unit tests.

Reply:
Regarding delimiters it's a good catch, the 'strict' is currently ignored on the 2.7. If you think it's needed I could add duplicity check in the post parse.